.travis.yml

```diff
@@ -64,231 +69,268 @@ jobs:
       env: DB=mysqli
 
   include:
-    - stage: Test
+    - stage: smoke testing
       php: 7.1
       env: DB=sqlite COVERAGE=yes
 
-    - stage: Test
+    - stage: smoke testing
+      if: type = pull_request
+      php: 7.1
+      env: DB= PHP_CodeSniffer=yes
```

Comment:
This is a purely cosmetic addition. Neither of the variable influence the build but are displayed in the job list and help to understand what the job is about.